src/site/content/en/blog/optimize-ttfb/index.md

```diff
@@ -161,18 +180,34 @@ CDN providers may also offer benefits beyond edge servers:
 
 While adopting a CDN involves a varying amount of effort from trivial to significant, it should be a high priority to pursue in optimizing your TTFB if your website is not already using one.
 
+### Used cached content where possible
+
+CDNs allow content to be cached at edge nodes, closer to visitors, if the content is configured with the appropriate [`Cache-Control` HTTP headers](https://developer.mozilla.org/docs/Web/HTTP/Headers/Cache-Control). While this is not appropriate for personalised content, requiring a trip all the way back to the origin can negate much of the value of a CDN.
+
+For sites that frequently update their content, even a short caching time can result in noticable performance gains for busy sites as only the first visitor during that time pays the full price, while all other visitors can use the cached resource. Some CDNs allow cache invalidation on site releases allowing the best of both worlds—long cache times, but insteant updates when needed.
```

Comment:
```suggestion
For sites that frequently update their content, even a short caching time can result in noticable performance gains for busy sites, since only the first visitor during that time experiences the fully latency back to the origin server, while all other visitors can reuse the cached resource from the edge server. Some CDNs allow cache invalidation on site releases allowing the best of both worlds—long cache times, but instant updates when needed.
```